hr_timesheet_sheet_week_start_day/README.rst

```diff
@@ -0,0 +1,71 @@
+.. image:: https://img.shields.io/badge/license-AGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl.html
+   :alt: License: AGPL-3
+
+=================================
+Hr Timesheet Sheet Week Start Day
```

Comment:
I think is better to keep the name of the module in line with the technical name.